src/third_party/koral/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+PROJECT(koral)
+cmake_minimum_required (VERSION 3.1)
+
+IF (${OpenMVG_USE_CUDA})
```

Comment:
use `if (OpenMVG_USE_CUDA)`

Please use lower case name for all the cmake functions